### ISSUE TITLE
Reduce frequency of alerts during finger dragging

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#455]
From #452, we are working on supporting grab/drag with mobile a11y (VoiceOver and Talkback). In the initial release of BASE we optimized alerts during drag so that they sound good during keyboard interaction.

Currently when moving with a finger we get too many alerts and stale alerts queue up. Improvements in utteranceQueue should make it pretty straight forward to reduce the number of alerts, I will investigate.

Tagging @terracoda since this is related to #452.

**Comment by terracoda** [comment]
@jessegreenberg, thanks for posting this issue. I noticed in my initial exploration in #452  last week that alerts were piling up when dragging. I was going to bring that up in our meeting today. 

Great news that the utteranceQueue is robust enough to delivery responses based on platform needs.

**Comment by terracoda** [comment]
@jessegreenberg, in #453 I mentioned I wasn't hearing "Release" responses. It's possible they are getting lost due tot eh pile up of responses in the utteranceQueue.

**Comment by jessegreenberg** [comment]
This is high priority as it is related to vibration work.

**Comment by terracoda** [comment]
@jessegreenberg, my first guess on pseudo logic would be to limit utterance to once or twice per region. Originally, we had several regions defined for all three areas: Sweater, Wall and Play Area. For keyboard navigation we have a certain default step size that usually results in about 1 to 2 mentions of a region as a learner is moving, and no more than 2 silent steps.

My guess (totally naive guess) is that something similar or even something less fine-grained will work for touch users.

**Comment by jessegreenberg** [comment]
@terracoda that makes sense. We could definitely explore that, but it would involve sim logic and take extra time.

Would you be OK with doing something time based? UtteranceQueue would support this without additional sim changes. The behavior would be something like "when dragging the balloon only alert movement changes once every 5 (or so) seconds".

**Comment by terracoda** [comment]
Using time might work, but the exploration is actually a spatial one. I think 5 seconds is likely too long. I'd start off with 3 seconds.

If we assume blind learners might drag somewhat slowly (be more careful about their dragging explorations) which is a big assumption on my part, 3 seconds might be a good place to start. 

I am also not assuming a default speech rate for the screen reader, but maybe something closer to 55 or 60.

**Comment by jessegreenberg** [comment]
Sounds good @terracoda, I went with that. In the above commit I reduced the frequency of alerts. I also made it so that small movements with a finger didn't generate any alerts. I did this because I found it difficult to hold my finger perfectly still on the the touch screen. So when using a finger or mouse a certain amount of movement is required to generate certain alerts.

@terracoda would you like to review this? If you don't have a touch screen you can get a sense of the output by playing with the a11y view with a mouse. Doesn't have to be done right away, I think this is working well enough now for what is needed for vibration. Removing high priority label.

> I found it difficult to hold my finger perfectly still on the the touch screen

This is my biggest concern at the moment, but it will apply to all of our drag and drop, not just BASE.

**Comment by terracoda** [comment]
@jessegreenberg, I have an iPhone 7. I'd love to try it out. I think a dev link should allow me to access this change.
